packages/block/test/block.spec.ts

```diff
@@ -1,18 +1,19 @@
-import * as tape from 'tape'
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
 import { NestedUint8Array, toBuffer, zeros } from '@ethereumjs/util'
 import { RLP } from 'rlp'
-import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import * as tape from 'tape'
+// eslint-disable-next-line import/order, @typescript-eslint/no-unused-vars
+import util from 'util'
+
+// explicitly import util, needed for karma-typescript bundling
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
```suggestion
// explicitly import util, needed for karma-typescript bundling
// eslint-disable-next-line import/order, @typescript-eslint/no-unused-vars
import util from 'util'
```
Also, why do we want to disable import/order for this one? Does having that import on top of the file break something?